src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -45,7 +45,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     </Otherwise>
   </Choose>
 
-  <Target Name="_CheckRuntimeIdentifier" BeforeTargets="_CheckForInvalidConfigurationAndPlatform">
+  <!-- TODO: change BeforeTargets="PrepareForBuild" back to BeforeTargets="_CheckForInvalidConfigurationAndPlatform"
+             once we update to a version of MSBuild where _CheckForInvalidConfigurationAndPlatform is not an initial target -->
+  <!-- With _CheckForInvalidConfigurationAndPlatform as an initial target, this can stop a NuGet restore from succeeding, which we don't want -->
+  <Target Name="_CheckRuntimeIdentifier" BeforeTargets="PrepareForBuild">
```

Comment:
I double checked, and in the version of the CLI we're using (1.0.0-preview3-003961), Microsoft.Common.CurrentVersion.targets has `InitialTargets="_CheckForInvalidConfigurationAndPlatform"`.